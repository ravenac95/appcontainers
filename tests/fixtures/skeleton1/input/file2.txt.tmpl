This file uses a template. The data in here will change. Here's the information
on the resource reservation from appcontainers:

ResourceReservation:
    - name: {{reservation.name}}
    - ip: {{reservation.ip}}
    - mac: {{reservation.mac}}
