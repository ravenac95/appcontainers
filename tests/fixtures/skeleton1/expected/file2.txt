This file uses a template. The data in here will change. Here's the information
on the resource reservation from appcontainers:

ResourceReservation:
    - name: SOMENAME
    - ip: 192.168.0.1
    - mac: 00:16:3e:00:00:01
